docs/building-apps/governing-apps/tokenomics/sns-checklist.mdx

```diff
@@ -110,14 +110,19 @@ In addition to performing comprehensive testing of all dapp operations using the
 The documentation prepared in sections 1.1 and 1.2 should be made publicly available ahead of the SNS launch. This provides transparency about the dapp, future plans with the dapp, the technical architecture, and the tokenomics. This information can be shared on the dapp’s website, GitHub, or wherever it would make sense.
 
 ## 3.2. Community discussion
-It’s strongly recommended that developers have a discussion in the forum with the community about the decentralization plans. It’s suggested to start a thread at least a couple of weeks before the NNS proposal is created. This will allow the community to learn about your plans, ask questions, and build trust in the SNS launch. The SNS initialization and decentralization swap will not start unless enough NNS neurons vote in favor of the proposal.
+It’s strongly recommended that developers have a discussion in the forum with the community about the decentralization plans. It’s suggested to start a thread at least a couple of weeks before the NNS proposal is created. Please use the [SNS DAO > SNS Launch Proposals](https://forum.dfinity.org/c/sns-dao/sns-launch-proposals) subcategory on the forum. This will allow the community to learn about your plans, ask questions, and build trust in the SNS launch. The SNS initialization and decentralization swap will not start unless enough NNS neurons vote in favor of the proposal.
 
 It is recommended to share:
 
-- [x] The init file that is planned to be used. You can start from a [template](https://github.com/dfinity/ic/blob/master/rs/sns/cli/test_sns_init_v2.yaml).
+- [x] The init file that is planned to be used. You can start from a [template](https://github.com/dfinity/ic/blob/master/rs/sns/cli/test_sns_init_v2.yaml). Make sure you add comments providing context for the values specified. E.g. elaborate on the specific dapp canisters or the distribution of developer neurons.
 - [x] The whitepaper with a full description of the decentralization and tokenomics.
 - [x] Technical decomposition of the dapp architecture in terms of canisters, source code, and documentation so that the community can validate that the dapp will actually be a decentralized application after the swap.
-- [x] Explanation of the extent to which security reviews were considered relevant for the dapp and what kind of security reviews have been conducted for the dapp.
+- [x] Explanation of the extent to which security reviews were considered relevant for the dapp and what kind of security reviews have been conducted for the dapp, core DAO functionality open-sourced.
```

Comment:
What do we mean by "core DAO funtionality"? In the context of SNSs, I would think here of the SNS cansiters, which are open sources without the team having to do anything. Therefore I assume you mean something else?
Maybe core dapp functionality? or custom DAO-functionality? Or we could even add a concrete example to make it clearer?

Reply:
This is reworded from our [voting guidelines](https://forum.dfinity.org/t/dfinitys-voting-on-upcoming-sns-launch-proposals/19543?u=cryptoschindler), but I agree with you. I like "core dapp functionality"